### DISCUSSION TITLE
Enable Audio for RTSP Stream to Scrypted and Homekit

### DISCUSSION THREAD

**Discussion by candecide** [mrlt8/docker-wyze-bridge#964] [Q&A] [answered]
I used this container to get the RTSP of my Cam Pan and through Scrypted I was able to use Homekit. It's been working well but so far I don't have audio to the stream. Looking at the wiki page on [Camera Audio](https://github.com/mrlt8/docker-wyze-bridge/wiki/Camera-Audio), it says I should enable it the ENV. Forgive my ignorance but what and where is ENV? And where to write the "- ENABLE_AUDIO=True" command to enable audio? I tried the terminal in Docker but it wasn't working. Thank you!

**Comment by mrlt8** [comment]
You would normally add that to your `docker-compose.yml`, but if you're using `docker run`, then you can just pass it as an extra environment variable `-e ENABLE_AUDIO=True`

```
docker run -p 8554:8554 -p 8888:8888 -p 5000:5000 -e ENABLE_AUDIO=True mrlt8/wyze-bridge
```

FYI, there's currently an issue with the audio lagging behind the video right now, which should hopefully be resolved soon.

**Reply by candecide** [reply]
Thanks. I am using the Mac Docker Desktop and I did not create the docker-compose.yml previously. I am lost on where to create the docker-compose file and or use your docker run command you provided. Can you provide further guidance on this? Sorry -- never used docker and a total newbie here.

**Reply by mrlt8** [reply, marked as answer]
You should be able to run `docker stop $(docker ps -q)` in terminal to stop the previous container, then use an updated `docker run` command with `-e ENABLE_AUDIO=True` to enable audio.

For docker compose, you can download the sample file from here: https://github.com/mrlt8/docker-wyze-bridge/blob/main/docker-compose.sample.yml, then make any necessary changes and start the container with `docker compose -f ` drag and drop your yml file to the terminal window then add `up` to the end of the line before running the command.

should look like this:
```
docker compose -f /some/path/docker-compose.sample.yml up
```

**Reply by candecide** [reply]
Got it running! I think I understand the two different approaches now. Thank you so much! I do notice the audio delay and hopefully it will be resolved soon.
